cache-config/tm-health-client/config/config.go

```diff
@@ -219,6 +224,9 @@ func GetTrafficMonitors(cfg *Cfg) error {
 
 	// login to traffic ops.
 	session, _, err := toclient.LoginWithAgent(cfg.TOUrl, cfg.TOUser, cfg.TOPass, true, "tm-health-client", false, GetRequestTimeout())
+	if err != nil {
+		return errors.New("could not establish a TrafficOps session: " + err.Error())
```

Comment:
constructing errors from errors should use `fmt.Errorf("... %w ...", err)` instead of `errors.New("... " + err.Error() + " ...")` so that the identity of the original error is not destroyed

Reply:
made the change